public/app/features/trails/MetricSelect/MetricSelectScene.tsx

```diff
@@ -28,13 +28,15 @@ import {
 } from '@grafana/scenes';
 import { Alert, Field, Icon, IconButton, InlineSwitch, Input, Select, Tooltip, useStyles2 } from '@grafana/ui';
 import { Trans } from 'app/core/internationalization';
+import { getSelectedScopes } from 'app/features/scopes';
 
 import { MetricScene } from '../MetricScene';
 import { StatusWrapper } from '../StatusWrapper';
 import { Node, Parser } from '../groop/parser';
 import { getMetricDescription } from '../helpers/MetricDatasourceHelper';
 import { reportExploreMetrics } from '../interactions';
-import { limitOtelMatchTerms } from '../otel/util';
+// TODO: fix this
+// import { limitOtelMatchTerms } from '../otel/util';
```

Comment:
@bfmatei Do you need support for this fix or just commented out temporarily?

Reply:
They were commented out temporarily as I couldn't test it. I'm revisiting this.

Reply:
Pushed update to fix it.